### PR TITLE
Update ALPN instructions.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

**Comment by deflaux** [comment]
See https://github.com/googlegenomics/dataflow-java/pull/160 for similar updates for dataflow.

File: README.md [hunk 1]
```diff
@@ -37,15 +37,26 @@ dependencies {
...
+```
+cd utils-java
+mvn package
+```
 
 If you wish to modify this code while also modifying a project that depends upon it (e.g., [dataflow-java](https://github.com/googlegenomics/dataflow-java)), just
 
   1. Build and install the jar to your local maven repository via `mvn install`.
-  2. Then update the pom.xml of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.  For example, make a change similar to [this](https://github.com/googlegenomics/dataflow-java/commit/17122798d579e593470c864868a46092f8795dfa) but only in your local copy of pom.xml open for editing.
+  2. Then update the `pom.xml` of the dependent project to temporarily depend upon the particular SNAPSHOT version of utils-java now installed in your local maven repository.  For example, make a change similar to [this](https://github.com/googlegenomics/dataflow-java/commit/17122798d579e593470c864868a46092f8795dfa) but only in your local copy of `pom.xml` open for editing.
```
Comment (mbookman) [inline README.md]: Can "open for editing" be trimmed from this sentence? It isn't clear to me what it adds.

**Comment by mbookman** [comment]
On small thing, otherwise LGTM.

**Comment by deflaux** [comment]
fixed, thanks!

**Pull request merged**